src/settings.ts

```diff
@@ -591,3 +591,15 @@ export const SCHEDULE_SHOW_DAYS_AHEAD = 7;
  * @see https://firebase.google.com/docs/firestore/query-data/queries#in_not-in_and_array-contains-any
  */
 export const FIRESTORE_QUERY_IN_ARRAY_MAX_ITEMS = 10;
+
+export const KeyboardShortcutKeys = {
+  chat: "c",
+  movement: {
+    down: ["down", "s"],
+    left: ["left", "a"],
+    right: ["right", "d"],
+    up: ["up", "w"],
+  },
+  schedule: "S",
+  search: "/",
+};
```

Comment:
Since we already use lowercase `s` for movement, it might be strange to use uppercase `S` for an entirely different feature.

I don't have a good idea off the top of my head, but can we change this to a different key binding please. Maybe `e` for events? (not sure if that is intuitive enough for people)

Reply:
As mentioned in another comment in this review, it would also be good to add the `esc` keybinding here, rather than being in a string.

Reply:
done in f79f6bc94dd969fc81493962b77da308cda7c557.  I will comment further in that original [thread](https://github.com/sparkletown/sparkle/pull/1416#discussion_r638465340)

Reply:
Re
> Since we already use lowercase s for movement, it might be strange to use uppercase S for an entirely different feature.

IMHO it is ok since matches `[S]chedule`. I don't mind changing it to whatever (e.g. `e`) if you insist though.